Documentation/UserTsconfig/Setup.rst

```diff
@@ -159,4 +173,6 @@ titleLen
     positive integer
 
 :aspect:`Description`
-    Max. Title Length
+    Maximum length of rendered record titles in the BE interface.
+    It's used in several places: page tree, edit masks, workspace module, etc.
+    Tip: to find out where exactly set it to a low number.
```

Comment:
```suggestion
    Tip: to find out where this setting is applied, set it to a low number.
```

It is better readable now, I think. I had to read it twice to understand. Maybe also use the admonition `.. tip::`?